### DISCUSSION TITLE
Sourcing Display

### DISCUSSION THREAD

**Discussion by bjoerns1983** [OMOTE-Community/OMOTE-Firmware#64] [General]
Hi there, 
can anyone point me in the right direction where to source a suitable display either from Aliexpress or any German/European shop?
In the End i would need 10pcs if the Prototype works out.

**Comment by KlausMu** [comment]
I ordered two from Alibaba https://www.alibaba.com/product-detail/High-Quality-240-3-rgb-320_1600408828330.html
42 EUR (21 EUR for one) to Germany, including Tax and Shipping.
Ordered at 2023/12/11, arrived at 2024/01/02 (22 days).

Shipping costs are a little bit random. It's cheaper to buy 2x2 than to buy 4.
TFT works well, no problems. Some encountered ghost touches with other displays.

I did not try:
https://www.amazon.de/Display-Development-Tools-2-8-TFT/dp/B0C1RFJFC7
https://www.adafruit.com/product/2770

On Aliexpress it seems not to be available.

**Reply by bjoerns1983** [reply]
Yes the shipping costs seem extremely wierd.
If i would buy 10 pcs the charge 130€ shipping costs althought the product only costs 70€.

If there is a Rev 2 of the Omote it would be great if it could feature a more standard display. You find many displays with the Ili9341 controllers  but most of them only have a 37 Pin connector on it.

**Reply by KlausMu** [reply]
Correct, but 20€ per display is not that bad.

**Reply by bjoerns1983** [reply]
Okay order two now so i can test the prototype also for 42€.
Hope they will arrive soon, normally no issue when i order from Aliexpress.

**Reply by KlausMu** [reply]
Experience will be a little bit different, but I had no problems. Alibaba is for B2B, Aliexpress for B2C.

**Comment by JustMe-NL** [comment]
You can also try digikey, orders above 50 euro are shipped for free to the EU and they take care of the import taxes.
